05_syntax_lab/04.py

```diff
@@ -1,8 +1,11 @@
-"""
-Write a program that reads lines from the user
-until an empty line is inserted.
-After the user typed in an empty line,
-print all previously inserted lines in reverse
-order (from last to first)
-"""
-
+print "Please enter any word"
+word = raw_input()
+total = word
+index = 0
+while word != ' ':
+	if index > 0:
+		total = "%s %s"  % (word, total)
+	print "Please add another word"
+	word = raw_input()
+	index = index + 1
+print total

```

Comment:
Please simplify this solution eliminating the two separate calls to ```raw_input``` and eliminating the ```index```.
BTW these are rows you read in not words and thus you need to add newline between them and not space.
You could also use ```+``` to concatenate two strings.